example/starwars/starwars.go

```diff
@@ -25,6 +25,7 @@ var Schema = `
 		character(id: ID!): Character
 		droid(id: ID!): Droid
 		human(id: ID!): Human
+		humans(height: Float): [Human]!
```

Comment:
`[Human!]!` would be better. Also I think it would be better to make this test separate from the Star Wars example for two reasons:
1. I expect there will be some more value coercion tests coming up.
2. I don't want to bloat the star wars schema with them, the schema should stay in line with what is used in the GraphQL documentation.